src/encoded/static/components/item-pages/MicroscopeConfigurationView.js

```diff
@@ -0,0 +1,552 @@
+                    + microConfTitleAppendStr + ' (' + copyCount + ')'  // Add new count
+                );
+            } else {
+                // Our title already has " - user's copy" substring, but not an " (int)"
+                microConfTitle = context.display_title + ' (2)';
+            }
+        }
+
+        const fallbackCallback = (errResp, xhr) => {
+            // Error callback
+            Alerts.queue({
+                'title': "Failed to save configuration.",
+                'message': "Sorry, can you try to save again?",
+                'style': 'danger'
+            });
+            this.setState({ 'cloneLoading': false });
+        };
+
+        microscope.Name = microConfTitle;
+
+        const payload = {
+            'title': microConfTitle,
+            'description': microConfDesc,
+            'microscope': microscope,
+            // We don't include other properties and let them come from schema default values.
+            // For example, default status is 'draft', which will be used.
+            // Lab and award do not carry over as current user might be from different lab.
+        };
+
+        // Try to POST/PUT a new viewconf.
+        this.setState(
+            { 'cloneLoading': true },
+            () => {
+                ajax.load(
+                    '/microscope-configurations/',
+                    (resp) => { // We're likely to get a status code of 201 - Created.
+                        this.setState({ 'cloneLoading': false }, () => {
+                            const newItemHref = object.itemUtil.atId(resp['@graph'][0]);
+
+                            // Redirect the user to the new Microscope display.
+                            navigate(newItemHref, {}, (resp) => {
+                                // Show alert on new Item page
+                                Alerts.queue({
+                                    'title': "Saved " + microConfTitle,
+                                    'message': "Saved new display.",
+                                    'style': 'success'
+                                });
+                            });
+                        });
+                    },
+                    'POST',
+                    fallbackCallback,
+                    JSON.stringify(payload)
+                );
+            }
+        );
+
+    }
+
+    /**
+      * Cancel/hide modal popup
+      * @returns {void}
+    */
+    handleModalCancel() {
+        this.setState({ 'modal': null });
+    }
+
+    /**
+        * Copies current URL to clipbard.
+        * Sets the microscope display status to released if it isn't already.
+        *
+        * @returns {void}
+    */
+    handleStatusChange(statusToSet = 'released', evt) {
+        evt.preventDefault();
+
+        const { context, href } = this.props;
+        const mtc = this.getMicroscopyMetadataToolComponent();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['mtc' is assigned a value but never used. (no-unused-vars)](https://app.codacy.com/app/4dn/fourfront/pullRequest?prid=4271272)